README.md

```diff
@@ -1276,51 +1280,15 @@ Knowing what's going on
 
         * [JBPM](https://issues.jboss.org/plugins/servlet/streams?key=JBPM&os_authType=basic)
 
-        * [GUVNOR](https://issues.jboss.org/plugins/servlet/streams?key=GUVNOR&os_authType=basic)
-
     * Subscribe to github repository commits:
 
         * [droolsjbpm-build-bootstrap](https://github.com/kiegroup/droolsjbpm-build-bootstrap/commits/master.atom)
 
-        * [droolsjbpm-knowledge](https://github.com/kiegroup/droolsjbpm-knowledge/commits/master.atom)
-
-        * [drools](https://github.com/kiegroup/drools/commits/master.atom)
-
-        * [optaplanner](https://github.com/kiegroup/optaplanner/commits/master.atom)
-
-        * [jbpm](https://github.com/kiegroup/jbpm/commits/master.atom)
-
-        * [droolsjbpm-integration](https://github.com/kiegroup/droolsjbpm-integration/commits/master.atom)
-
-        * [guvnor](https://github.com/kiegroup/guvnor/commits/master.atom)
-
-        * [droolsjbpm-tools](https://github.com/kiegroup/droolsjbpm-tools/commits/master.atom)
-
-        * [droolsjbpm-build-bootstrap](https://github.com/kiegroup/droolsjbpm-build-bootstrap/commits/master.atom)
-
-    * Subscribe to [Jenkins](https://hudson.jboss.org/hudson/view/Drools%20jBPM/)
-
-        * with [the Firefox plugin](https://addons.mozilla.org/en-us/firefox/addon/jenkins-build-monitor/) to easily see in your status bar which builds are failing (recommended):
-
-            * After installation, right click on the Jenkins icon in the lower right corner.
-
-            * Click menu item *Preferences*, tab *Feed*, textfield *poll interval* `30` *minutes*.
-
-            * Click menu item *Preferences*, tab *Display*, combox *Display* `latest build` *on status bar*.
-
-            * Go to the Jenkins job of the projects you're working on:
-
-                * [guvnor](https://hudson.jboss.org/hudson/view/Drools%20jBPM/job/guvnor/)
-
-            * Right click in the lower left corner on the *All* feed link, menu item *Add link to Jenkins build monitor*.
-
-        * Otherwise, check [the Jenkins website](https://hudson.jboss.org/hudson/view/Drools%20jBPM/) often.
-
-            * Note: the public Jenkins is a mirror of the VPN internal Red Hat Jenkins and is sometimes stale.
+        * <kiegroup repository> i.e. https://github.com/kiegroup/<repository>/commits/master.atom
```

Comment:
as an example this should be exact URL to check like https://github.com/kiegroup/droolsjbpm-build-bootstrap/commits/master.atom

Reply:
@mareknovotny  Look at the example above - this is the droolsjbpm-build-bootstrap example. The line starting with <kiegroup repository> should be an example how to construct the URL. You see the line below - it links you to the file which reps we have in our kiegroup.
Maybe is is not clear that this is like an example. I didn't want to repeat 25 the same - only changing the rep name.

Reply:
@mareknovotny 
Better explained now